instrumentation/spring/spring-batch-3.0/javaagent/src/test/groovy/SpringBatchTest.groovy

```diff
@@ -78,3 +79,28 @@ class JavaConfigBatchJobTest extends SpringBatchTest implements ApplicationConfi
     new AnnotationConfigApplicationContext(SpringBatchApplication)
   }
 }
+
+class XmlConfigBatchJobTest extends SpringBatchTest implements ApplicationConfigTrait {
+  static final Config PREVIOUS_CONFIG = updateConfig {
```

Comment:
Do you think this can benefit from the similar change? https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1915

Reply:
I've tried it but it won't work 😢 

Probably because `otel.instrumentation.spring-batch.enabled` is accessed when `AgentTestRunner` setup methods run, and they run first - before `SpringBatchTest`'s setup.